### PR TITLE
Fix alternate opening time closed all day bug

### PR DESCRIPTION
**Pull request by markysoft**
Set 'opens' and 'closes' to undefined on openingTimesAlterationsAsOffset when closed all day

### CODE REVIEW

File: lib/es/importDataFromFile.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const config = require('../../config/config');
 const esConfig = require('../../config/esConfig');
 
 function getDateStamp(date) {
-  return moment(date).format('YYYYMMDDHHmmSS');
+  return moment(date).format('YYYYMMDDHHmmss');
```
Comment (markysoft) [inline lib/es/importDataFromFile.js:10]: changed fractional seconds `S` to seconds `s` in datestamp

File: config/pharmacies/transform.js [hunk 2]
```diff
@@ -21,6 +21,10 @@ function timesToMinutesSinceSunday(day, index) {
 }
 
 function altTimesToMinutesSinceMidnight(date, sessions) {
+  if (sessions.length === 0) {
+    // mark as closed all day
+    return { date, opens: undefined, closes: undefined };
```
Comment (st3v3nhunt) [inline config/pharmacies/transform.js]: Could this not just be `{ date }`?

Comment (markysoft) [reply config/pharmacies/transform.js]: it could be, I thought it more specific to show the fields are intentionally undefined. Happy to change it.

Comment (st3v3nhunt) [reply config/pharmacies/transform.js]: I thought that was what we'd agreed yesterday i.e. to leave it blank. The way that decision was reached was from thinking about using numbers e.g. 0, -1, 3600 or some other large value all of which are a bit of bodge. To using nothing that is easier to reason about than `undefined` due to the model also supporting the lack of the `date` key in `openingTimesAsOffset`.

Comment (markysoft) [reply config/pharmacies/transform.js]: amounts to the same thing, I'll set it to just be `{date}` so it looks the same as the json returned from an ES query

File: test/unit/pharmacies/transform.js [hunk 3]
```diff
@@ -147,12 +148,13 @@ describe('Profiles transform', () => {
...
     // eslint-disable-next-line
     expect(otAltOffset).to.exist;
-    expect(otAltOffset.length).to.equal(5);
+    expect(otAltOffset.length).to.equal(6);
     expectAltOpenClose(otAltOffset[0], '2014-05-05', 660, 960);
     expectAltOpenClose(otAltOffset[1], '2014-05-26', 660, 960);
     expectAltOpenClose(otAltOffset[2], '2014-04-18', 660, 960);
     expectAltOpenClose(otAltOffset[3], '2014-04-21', 660, 960);
     expectAltOpenClose(otAltOffset[4], '2014-08-25', 660, 960);
+    expectAltOpenClose(otAltOffset[5], '2015-01-01', undefined, undefined);
```
Comment (st3v3nhunt) [inline test/unit/pharmacies/transform.js]: Can a specific test be added for this scenario please? In order to make it clear this is what represents the data for a closed alteration? I think by doing that the comment in `transform.js` could be removed.

**Review by st3v3nhunt — APPROVED** [review]
LGTM 👍

**Pull request merged**